src/test/java/uk/ac/vfb/geppetto/test/PartsOfAdultBrainTest.java

```diff
@@ -190,18 +202,21 @@ public void testRunQuery() throws GeppettoInitializationException, GeppettoVisit
 
 		Variable variable = geppettoModelAccess.getPointer("FBbt_00003624").getElements().get(0).getVariable();
 
-		int count = aberDataSource.getNumberOfResults(getRunnableQueries(model.getQueries().get(avQ.get("partsof")), variable));
-		Assert.assertEquals(1626, count);
-
-		QueryResults results = aberDataSource.execute(getRunnableQueries(model.getQueries().get(avQ.get("partsof")), variable));
-
-		Assert.assertEquals("ID", results.getHeader().get(0));
-		Assert.assertEquals("Name", results.getHeader().get(1));
-		Assert.assertEquals("Definition", results.getHeader().get(2));
-		Assert.assertEquals("Type", results.getHeader().get(3));
-		Assert.assertEquals("Images", results.getHeader().get(4));
-		Assert.assertEquals(1626, results.getResults().size());
-
+		int countOWL = owleryDataSource.getNumberOfResults(getRunnableQueries(model.getQueries().get(avQ.get("partsof")), variable));
+		try{
+			Assert.assertTrue(1600<countOWL);
```

Comment:
@ddelpiano ditto regarding the try/catch and the more "permissive" check

Reply:
@tarelli again, as Rob said the number can grow, so we can check that at least we are expecting a number "bigger than".